pyproject.toml

```diff
@@ -32,6 +32,12 @@ dependencies = [
 mace = [
     "janus-core[mace]"
 ]
+qe = [
+    "aiida-quantumespresso<4.13, >=4.12",
+]
+fpsample = [
+    "fpsample<0.4, >=0.3.3",
+]
```

Comment:
Is this not a required dependency?

Reply:
For this fine-tuning, yes, but so is mace for several of the tutorials. I wouldn't say it's fundamental to aiida-mlip, but probably worth saying at the start of the tutorial what optional extras we use?